.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: python
 
+services:
+  - docker
+
 python:
   - "3.6"
 
 script:
   - python setup.py flake8
+  - cp docker/.env.example docker/.env
+  - docker build . -t pyslackers/sirbot.pythondev -f docker/Dockerfile
+  - docker-compose -f docker/docker-compose.ngrok.yml build
+  - docker-compose -f docker/docker-compose.yml build
```

Comment:
I prefer to keep them to be sure the file are correct. But indeed I could replace that by https://docs.docker.com/compose/reference/config/